less/circle-background.less

```diff
@@ -0,0 +1,164 @@
+@circle-bg-xs-diameter: 25%;
+@circle-bg-sm-diameter: 50%;
+@circle-bg-md-diameter: 120%;
+@circle-bg-lg-diameter: 200%;
+@circle-bg-xl-diameter: 400%;
+@circle-bg-translate-to-edge: 5%;
+@circle-bg-translate-to-center:50%;
+@circle-bg-translate-to-center-asymmetric:35%;
+
+.circle-bg {
+  position: relative;
+  z-index: 1;
+  &:before {
+    content: "";
+    width: @circle-bg-md-diameter;
+    padding-bottom: @circle-bg-md-diameter;
+    border-radius: 500rem;
+    position: absolute;
+    background: @gray-base;
+    z-index: -1;
+    top: @circle-bg-translate-to-center;
+    left: @circle-bg-translate-to-center;
+    transform: translate(-@circle-bg-translate-to-center, -@circle-bg-translate-to-center);
+  }
+}
+
+/*Sizes*/
+
+.circle-bg-xs:before {
+  .circle-bg-variant(@circle-bg-xs-diameter,@circle-bg-xs-diameter);
```

Comment:
Could we change this mixin to `.circle-bg-size()`.  I could see us using other mixins for colour so more specific is better.  Does it need two parameters, they look like they're always the same?